docs/tutorials/cargo_tests/buffer/Makefile

```diff
@@ -6,14 +6,16 @@ export CARGO_TARGET_DIR ?= target
 
 all: buffer_data.postcard
 
+FORCE:
+
 bin/icu4x-datagen:
 	echo "Target dir: $(CARGO_TARGET_DIR)"
 	cargo install icu_datagen --root .
 
-bin/demo_buffer: Cargo.toml src/*.rs
+bin/demo_buffer: FORCE
 	echo "Target dir: $(CARGO_TARGET_DIR)"
-	cargo build --release
-	cp "$(CARGO_TARGET_DIR)"/release/demo_buffer bin/demo_buffer
+	echo "If the following command fails, run cargo build --release first."
```

Comment:
That's not very makefile-y.

`ls "$(CARGO_TARGET_DIR)"/release/demo_buffer || cargo build --release`

Reply:
eh

Reply:
Added more comments